cmd/use.go

```diff
@@ -0,0 +1,22 @@
+package cmd
+
+import (
+	"github.com/gravitl/netclient/functions"
+	"github.com/spf13/cobra"
+)
+
+// useCmd represents the use command
+var useCmd = &cobra.Command{
+	Use:   "use [version]",
```

Comment:
square brackets means the argument is optional right? but it seems there isn't a default value so maybe remove the square brackets

Reply:
done